homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -0,0 +1,56 @@
+"""Platform for Husqvarna Automower base entity."""
+
+import logging
+from typing import Any
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import AutomowerDataUpdateCoordinator
+from .const import DOMAIN, HUSQVARNA_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AutomowerEntity(CoordinatorEntity[AutomowerDataUpdateCoordinator]):
+    """Defining the Automower base Entity."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    def __init__(self, coordinator: AutomowerDataUpdateCoordinator, idx: int) -> None:
+        """Initialize AutomowerEntity."""
+        super().__init__(coordinator, context=idx)
```

Comment:
Where is the context needed?

Reply:
Don't know, where this is coming from. I've removed it.